changes/2029.removal.8.rst

```diff
@@ -1 +1 @@
-The ``tight`` argument has been removed from ``Canvas.measure_text()``. It can be removed without impact.
+The ``tight`` argument to ``Canvas.measure_text()`` has been deprecated. It was never implemented, and can be safely removed without impact.
```

Comment:
Not entirely true - it was implemented on GTK (surfacing a platform implementation detail).

Reply:
OK, I'll reword it to match the note on `preserve`.